src/redux/epics/getDataPendingEpic.js

```diff
@@ -0,0 +1,33 @@
+import { buildUrl } from '../../shared/buildUrl'
+import { actions as dataSliceActions } from '../data/dataSlice'
```

Comment:
meglio esportare direttamente `dataSliceActions` in modo da sfruttare auto import e non dover sempre assegnare un alias